Library/Formula/glslang.rb

```diff
@@ -0,0 +1,21 @@
+class Glslang < Formula
+  desc "glslang is an OpenGL and OpenGL ES shader front end and validator."
+  homepage "https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/"
+  url "https://github.com/KhronosGroup/glslang/archive/3.0.tar.gz"
+  sha256 "91653d09a90440a0bc35aa490d0c44973501257577451d4c445b2df5e78d118c"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+      bin.install "StandAlone/glslangValidator"
```

Comment:
@bfontaine The ```CMAKE_INSTALL_PREFIX``` variable isn't overridden in this release of glslang, but has been fixed for future releases: https://github.com/KhronosGroup/glslang/commit/4cbf748b133aef3e2532b9970d7365304347117a